.gitignore

```diff
@@ -88,6 +88,25 @@ env/
 venv/
 ENV/
 
+# Vim stuff
+[._]*.s[a-v][a-z]
+!*.svg  # comment out if you don't need vector files
+[._]*.sw[a-p]
+[._]s[a-rt-v][a-z]
+[._]ss[a-gi-z]
+[._]sw[a-p]
+
+# Session
+Session.vim
+Sessionx.vim
+
+# Temporary
+.netrwhist
+*~
+# Auto-generated tag files
+tags
+# Persistent undo
+[._]*.un~
```

Comment:
Please, rebase your changes in the latest version. There are changes adding some of the same stuff.